test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ControllerUnitTestabilityTests.cs

```diff
@@ -688,22 +688,22 @@ public IActionResult CreatedAtRoute_Action(string routeName, object routeValues,
 
             public IActionResult HttpBadRequest_Action()
```

Comment:
Suspect @Eilon wants you to ignore me here (see https://github.com/aspnet/Mvc/pull/4107#discussion-diff-53069339).